requirements.dev.txt

```diff
@@ -1,5 +1,5 @@
 awscli==1.16.283
-git+git://github.com/DataBiosphere/azul-chalice@1.14.0+4#egg=chalice
+git+git://github.com/DataBiosphere/azul-chalice@1.14.0+5#egg=chalice
```

Comment:
`make requirements_update` needs to be run and the resulting changes committed

Reply:
Nothing changed in the `*.trans.txt` files when running `make requirements_update`

Reply:
Repeat before every review.